### ISSUE TITLE
Feature: новые пользователи по ссылкам /user/username видят /intro/username вместо страницы приветствия

### ISSUE THREAD

**Issue by psydvl** [vas3k/vas3k.club#618]
<details>
<summary>Предисловие</summary>
Я только зарегистрировался и не написал intro. Как пример показаны только три.<br>
Поэтому я пошел по комментариям почитать intro других пользователей.<br>
Однако при нажатии на пользователей меня перекидывает на страницу создания моего интро.<br>
Пришлось все же набрать ручками <code>/intro/[username]</code>, что довольно муторно😞<br>
</details>

Идея: при нажатии на ссылку пользователя у тех, кто еще без intro делать с `/user/[username]` переадресацию на `/intro/[username]` вместо `/intro`

**Comment by devcooch** [comment]
Правильно, потому что пока ты не написал интро, тебя не берут в клуб, а значит смотреть закрытые страницы (==профили других пользователей) низзя.

**Comment by psydvl** [comment]
Еще раз
Все страницы вида /intro/[username] открыты и сейчас, хотя я не отправлял свое intro, но приходится переходить на них ручками.
Или это баг?

**Comment by devcooch** [comment]
Давай посмотрим, что Вастрик скажет.

**Comment by vas3k** [comment]
Открыты вроде бы только те страницы, которые авторы согласились открыть. Если нет, то это баг

**Comment by psydvl** [comment]
Так я все же не об этом.   
На примерах покажу: открываем, например, `https://vas3k.club/intro/vas3k/` и смотрим, первый комментарий от Gleb Markin, ссылка под именем ведет на `https://vas3k.club/user/captaingim/`, но меня как новичка она переадресует на `https://vas3k.club/intro/`.  
А хочется на `https://vas3k.club/intro/captaingim/`, чтобы посмотреть на интро.
И я вбиваю это ручками...

**Comment by vas3k** [comment]
Вбиваешь ручками и открывается?

**Comment by psydvl** [comment]
Да

**Comment by vas3k** [comment]
Хм, я проверю. Обычно открываться не должно, толко если юзер явно изъявил желание быть открытым для всех

**Comment by psydvl** [comment]
Добавлю, потому что все же немного запутался:
1. Если новые пользователи могут видеть только три интро, ровно те что показаны как примеры, надо переназвать Issue в Bug
2. Если новые пользователи могут посмотреть интро и других пользователей (даже если только тех кто разрешил), то тогда это Feature и хотелось бы как-то более удобно чем ручками вбивая URL получать к ним доступ.

**Comment by skywinder** [comment]
можно провреить, а где юзер указывает "открытость" профиля?

**Comment by vas3k** [comment]
> можно провреить, а где юзер указывает "открытость" профиля?

Нигде, это было сделано руками по договоренности с некоторыми юзерами (мной, например)
